apps/glusterfs/device_entry_test.go

```diff
@@ -1066,6 +1066,15 @@ func TestDeviceSetStateFailedWithEmptyPathBricks(t *testing.T) {
 
 // See also RHBZ#1572661
 func TestDeviceRemoveSizeAccounting(t *testing.T) {
+	t.Run("standard", func(t *testing.T) {
+		testDeviceRemoveSizeAccounting(t, false)
+	})
+	t.Run("arbiter", func(t *testing.T) {
+		testDeviceRemoveSizeAccounting(t, true)
+	})
+}
+
+func testDeviceRemoveSizeAccounting(t *testing.T, useArbiter bool) {
```

Comment:
Why does this function has the same name, just not exported? Does that play a role in the test framework?

Reply:
note also t vs T